browser/p3a/p3a_core_metrics.h

```diff
@@ -63,6 +64,29 @@ class BraveWindowTracker : public BrowserListObserver {
   DISALLOW_COPY_AND_ASSIGN(BraveWindowTracker);
 };
 
+class BraveNewTabCountTracker
```

Comment:
I'm curious this tracker should be in chrome layer.
If this is added p3a component layer, we can directly call `BraveNewTabCountTracker` method direclty from `ViewCounterService`. WDYT? cc @iefremov

Reply:
Instead of doing an observer, I could just put the code directly into `ViewCounterService`. Is that what you're proposing? I think that might be a good solution

Reply:
Updated! 😄 Thanks for the help over DM